### PR TITLE
Introduce semantic release

### PR DESCRIPTION
**Pull request by staust**
This introduce automatic semantically releases to the master branch. 
Manual Releases could be done by "npx semantic-release --no-ci". May we document this some where a its not documented in the tools help section?

### CODE REVIEW

File: AUTHORS.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Authors ordered by first contribution
 
 Stefan Staude - https://github.com/staust
+Vitaly Ivanov - ttps://github.com/viviivanov
```
Comment (Ninerian) [inline AUTHORS.txt]: Typo, https

**Comment by staust** [comment]
There is nothing to test

**Comment by staust** [comment]
this is a test comment for the slack integration

**Pull request merged**